meiki_server/notes/routes.go

```diff
@@ -144,5 +175,6 @@ func CreateRoutes(router *gin.RouterGroup, ctx context.Context, ns NotesStore) {
 	router.GET("/list", getListHandler(ctx, ns))
 	router.GET("/read/:id", getReadHandler(ctx, ns))
 	router.POST("/update/:id", getUpdateHandler(ctx, ns))
-	router.POST("/delete/:id", getDeleteHandler(ctx, ns))
+	router.POST("/rename/:id", genRenameHandler(ctx, ns))
```

Comment:
update/rename should be PUT, I think (https://www.mscharhag.com/api-design/http-post-put-patch)

Reply:
yep